src/main/java/step1/qna/domain/Answer.java

```diff
@@ -0,0 +1,76 @@
+package step1.qna.domain;
+
+import step1.qna.CannotDeleteException;
+import step1.qna.NotFoundException;
+import step1.qna.UnAuthorizedException;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+public class Answer extends AbstractEntity {
+	private static final String EXCEPTION_MSG = "다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.";
+	@ManyToOne(optional = false)
+	@JoinColumn(foreignKey = @ForeignKey(name = "fk_answer_writer"))
+	private User writer;
+
+	@ManyToOne(optional = false)
+	@JoinColumn(foreignKey = @ForeignKey(name = "fk_answer_to_question"))
+	private Question question;
+
+	@Lob
+	private String contents;
+
+	private boolean deleted = false;
+
+	public Answer() {
+	}
+
+	public Answer(User writer, Question question, String contents) {
+		this(null, writer, question, contents);
+	}
+
+	public Answer(Long id, User writer, Question question, String contents) {
+		super(id);
+
+		if (writer == null) {
+			throw new UnAuthorizedException();
+		}
+
+		if (question == null) {
+			throw new NotFoundException();
+		}
+
+		this.writer = writer;
+		this.question = question;
+		this.contents = contents;
+	}
+
+	public boolean isDeleted() {
+		return deleted;
+	}
+
+	public void checkDeleteCondition(User loginUser) throws CannotDeleteException {
+		if(!this.writer.equals(loginUser)) {
```

Comment:
```suggestion
		if (!this.writer.equals(loginUser)) {
```

컨벤션 확인해 주시고 포맷팅 단축키도 학습해 보시면 좋을 것 같아요~